models/models.js

```diff
@@ -24,14 +24,45 @@ const getArticlesByID = (id) => {
     )
     .then((data) => {
       if (!data.rows.length || data.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: 'Not found' });
+        return Promise.reject({ status: 404, msg: "Not found" });
       }
       return data.rows[0];
     });
 };
 
+const getAllArticles = () => {
+  return db
+    .query(
+      `
+  SELECT 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at, 
+  articles.votes, 
+  articles.article_img_url, 
+  COUNT(comments.comment_id) AS comment_count 
+  FROM articles 
+  LEFT JOIN comments ON articles.article_id = comments.article_id 
+  GROUP BY 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at, 
+  articles.votes, 
+  articles.article_img_url 
```

Comment:
This group by can definitely be simplified and does not need all of these column names.